src/renderers/shared/fiber/isomorphic/ReactCoroutine.js

```diff
@@ -42,7 +42,9 @@ export type ReactYield = {
   continuation: mixed
 };
 
-exports.createCoroutine = function<T>(children : mixed, handler : CoroutineHandler<T>, props : T, key : ?string = null) : ReactCoroutine {
+exports.createCoroutine = function<T>(
```

Comment:
Yes, no problem. To avoid an extra commit, will it work for me to amend the commit and force push to my fork? Or I can make a new pull request. What do you prefer?